docs/reference/authentication.md

```diff
@@ -27,6 +27,17 @@ Web UI at Administration / Users to create users.  Users have an access key
 `AKIA...` and an associated secret access key.  These credentials are valid
 to log into the Web UI, or to authenticate programmatic requests to the API
 Server or the S3 Gateway.
+git
+#### lakeFS Invite
```

Comment:
Something is a bit off with this title.  Perhaps

```suggestion
#### Invited to lakeFS?
```

@adipolak - can you help?

Reply:
Not sure I agree on this one. I think @adipolak should decide on this one